guestlink.php

```diff
@@ -63,7 +63,34 @@
     ];
 
     echo $OUTPUT->header();
+
+    // Setup access policy modal window
+    // If global editing is disabled, it is always the global version.
+    // Note: Duplicate block @ mod/bigbluebuttonbn/viewlib.php:258
+    if ((int) $CFG->bigbluebuttonbn_accessmodal_editable) {
+        $policytext = format_text($bbbsession['bigbluebuttonbn']->accesspolicy ?? '');
```

Comment:
Im a little hesitant about the null coalesce, iirc it breaks php 7.0. Can you just check the supported versions for this branch, and see if php 7.0 is in the mix?

Reply:
Unless this is a copy of my code that already uses it. in which case, I already broke it :)

Reply:
I'll check. I was about to add a different implementation, but this was actually code which already exists in another file mentioned above.

Reply:
Yeah thats my code then, dont worry about it :)